doc/ref_model/chapters/glossary.md

```diff
@@ -73,7 +74,23 @@ To help guide the reader, this glossary provides an introduction to the terminol
 - **Service Assurance (SA)**: collects alarm and monitoring data. Applications within SA or interfacing with SA can then use this data for fault correlation, root cause analysis, service impact analysis, SLA management, security, monitoring and analytic, etc.
 
 <a name="11.1.4"></a>
-### 1.1.4 Other Referenced Terminology
+### 11.1.4 Container Related Terminology
+
+>_*Note:*_ Relevant terms are added here from RA2. Most of these term definitions are taken from [Kubernetes glossary](https://kubernetes.io/docs/reference/glossary) but in some cases should be made independent from Kubernetes as a specific container orchestration engine.
+
+- **Container Image**:	Stored instance of a container that holds a set of software needed to run an application.
+- **Container**:	A lightweight and portable executable image that contains software and all of its dependencies.
```

Comment:
OCI defines [Conatiner](https://github.com/opencontainers/runtime-spec/blob/master/glossary.md) as "An environment for executing processes with configurable isolation and resource limitations. For example, namespaces, resource limits, and mounts are all part of the container environment."
Can we include it as a "Note" say.